system/lib/libc/musl/src/signal/setitimer.c

```diff
@@ -4,8 +4,72 @@
 
 #define IS32BIT(x) !((x)+0x80000000ULL>>32)
 
+#ifdef __EMSCRIPTEN__
+#include <signal.h>
+#include <stdio.h>
+#include <emscripten/emscripten.h>
+static double _emscripten_timers[3];
+static double _emscripten_intervals[3];
```

Comment:
Perhaps add a comment as to what these two are? The names themselves don't explain enough to me, so the reader needs to start looking at how they are used.

Reply:
Done, and added some comments.